src/pages/nft/class/_classId/index.vue

```diff
@@ -92,45 +107,68 @@
-                preset="h3"
-                align="center"
-                class="text-like-green mt-[38px] mb-[24px]"
+                @click="() => handleSwitchToCollection(false)"
               />
-              <ul
-                class="flex flex-wrap items-start justify-center gap-[24px] w-full"
-              >
-                <li
-                  v-for="editionConfig in nftEditions"
-                  :key="editionConfig.name"
-                >
-                  <NFTBookEditionCompareTableColumn
-                    class="w-[280px]"
-                    :src="NFTImageUrl"
-                    :edition-config="editionConfig"
-                    :class-id="classId"
-                    @click-collect="handleCollectFromEditionCompareTable"
-                  />
-                </li>
-              </ul>
-            </div>
-            <div v-if="nftCollections?.length" class="flex-col">
-              <Label
+              <div
+                v-if="nftCollections?.length"
+                class="w-[1px] bg-medium-gray h-[24px]"
+              />
+              <ButtonV2
+                v-if="nftCollections?.length"
+                :content-class="[
+                  'text-[28px]',
+                  'font-600',
+                  'text-medium-gray',
+                  { 'text-like-collection': shouldShowCollectionItem },
+                ]"
+                preset="plain"
                 :text="$t('nft_collection_label')"
-                preset="h3"
-                align="center"
-                class="text-like-green mt-[38px] mb-[24px]"
+                @click="() => handleSwitchToCollection(true)"
               />
-              <ul
-                class="flex flex-wrap items-start justify-center gap-[24px] w-full"
+            </div>
+            <ul
+              v-if="!shouldShowCollectionItem"
+              class="flex flex-wrap items-start justify-center gap-[16px] w-full"
+            >
+              <li
+                v-for="editionConfig in nftEditions"
+                :key="editionConfig.name"
               >
-                <li v-for="collection in nftCollections" :key="collection.id">
+                <NFTBookEditionCompareTableColumn
+                  class="w-[280px]"
+                  :src="NFTImageUrl"
+                  :edition-config="editionConfig"
+                  :class-id="classId"
+                  @click-collect="handleCollectFromEditionCompareTable"
+                />
+              </li>
+            </ul>
+            <ul
+              v-else
+              class="flex flex-wrap items-start justify-center gap-[24px] w-full"
+            >
+              <li v-for="collection in nftCollections" :key="collection.id">
+                <NuxtLink
+                  :to="
+                    localeLocation({
+                      name: 'nft-collection-collectionId',
+                      params: { collectionId: nftCollections[0].id },
```

Comment:
```suggestion
                      params: { collectionId: collection.id },
```
We are in `v-for` with `collection` as item right?

Reply:
[🎨 Use collectionId instead of nftCollections[0]](https://github.com/likecoin/liker-land/pull/1516/commits/0085025ad3291412980baa29a606b9890f1daf47)